src/components/voltage-init-result.js

```diff
@@ -199,6 +199,48 @@ const VoltageInitResult = ({ result, status, tabIndex, setTabIndex }) => {
         ];
     }, []);
 
+    function renderTotalReactiveSlacks(reactiveSlacks) {
+        const totalInjection = resultToShow.reactiveSlacks
+            ? resultToShow.reactiveSlacks
+                  .filter((reactiveSlack) => reactiveSlack.slack < 0)
+                  .reduce((sum, reactiveSlack) => sum + reactiveSlack.slack, 0)
+            : 0;
+        const totalConsumption = resultToShow.reactiveSlacks
+            ? resultToShow.reactiveSlacks
+                  .filter((reactiveSlack) => reactiveSlack.slack > 0)
+                  .reduce((sum, reactiveSlack) => sum + reactiveSlack.slack, 0)
+            : 0;
```

Comment:
You can do more refactoring here.
You are practically using the same code to get totalInjection and totalConsumption.